### DISCUSSION TITLE
Cell coloring [Aug 15]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#6211] [Discord Threads]
> 🙋‍♂️ **funapple**

**Comment by o1lab** [comment]
> 👨‍🦲 **dstala**  
#### not in immediate roadmap  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1129401173359726604)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
